Saucier720-app/src/app/deals/FEC/deals-store-button/deals-store-button.component.scss

```diff
@@ -35,4 +35,14 @@
   button:hover {
       background-color: var(--button-hover-color);
       color: var(--button-hover-text-color);
+    }
+
+  button:focus{
+    background-color: green;
+    color: white;
+  }
+
+  button.clicked {
+    background-color: green;
```

Comment:
i like it green, but would it be better to do something in the color palate?

Reply:
Up to you tho, i do like it green as well

Reply:
Could maybe make the green match the background and the text the yellowish?

Reply:
<img width="960" alt="newcolor" src="https://github.com/TylerMetz/Saucier720/assets/92872780/02a8ae5b-b97e-481b-b26d-263fb8a0b224">